pramen/core/src/main/scala/za/co/absa/pramen/core/pipeline/OperationDef.scala

```diff
@@ -125,6 +132,22 @@ object OperationDef {
       extraOptions))
   }
 
+  private def getThreadsToConsume(operationName: String, config: Config, appConfig: Config): Int = {
+    val maxThreads = appConfig.getInt(Keys.PARALLEL_TASKS)
+    val consumeThreads = ConfigUtils.getOptionInt(config, CONSUME_THREADS_KEY).getOrElse(DEFAULT_CONSUME_THREADS)
+
+    if (consumeThreads < 0) {
```

Comment:
`<= 0` ?

Reply:
ah, crap, thank you for spotting this! 😄